src/loader.rs

```diff
@@ -30,8 +30,7 @@ impl PixbufLoader {
             let ptr = ffi::gdk_pixbuf_loader_new_with_type(image_type.to_glib_none().0, &mut error);
             if error.is_null() {
                 Ok(from_glib_full(ptr))
-            }
-            else {
+            } else {
```

Comment:
I suppose we could try adopting rustfmt to handle that, but in this commit it's just unwelcome noise.